sunpy/net/vso/vso.py

```diff
@@ -888,3 +891,35 @@ def unknown_method(self, response):
     @classmethod
     def _can_handle_query(cls, *query):
         return all([x.__class__.__name__ in attrs.__all__ for x in query])
+
+    @classmethod
+    def register_values(cls):
+        try:
+            adict = cls.get_vso_values()
+        except URLError:
+            log.warn('Looks like the VSO server is not reachable.')
+            adict = {}
+        return adict
+
+    def get_vso_values():
```

Comment:
This needs rewriting to use the other VSO API I was recommended to use by the VSO team.

Reply:
I could not get it to work with requests so I gave up.

Reply:
oh probably form encoding, I had that issue.